hail/src/main/scala/is/hail/annotations/RegionValueBuilder.scala

```diff
@@ -433,6 +433,14 @@ class RegionValueBuilder(var region: Region) {
                 i += 1
               }
               endArray()
+            case is: Array[Annotation] =>
```

Comment:
Whoa, I don't think we can do this easily.  Whoever is producing  this array needs to turn it into an IndexedSeq.  Lots of code will break with this.